jpf-symbc/build.gradle

```diff
@@ -0,0 +1,192 @@
+apply plugin: "java"
+apply plugin: "maven-publish"
+
+def getCurrentVersion() {
```

Comment:
remove hardcoded version and use something similar to JPF: "JPF-8.0-112-ge91ea1f-SNAPSHOT"

Reply:
Hi @yannicnoller
I have gone through the build.gradle file of the jpf-core but have'nt figureout how they are using git-hash and java version to deploy it locally and I'll contact Mr.Cyrill regarding this.